GitHubTeams.ps1

```diff
@@ -150,11 +158,41 @@ filter Get-GitHubTeam
         'AccessToken' = $AccessToken
         'TelemetryEventName' = $MyInvocation.MyCommand.Name
         'TelemetryProperties' = $telemetryProperties
-        'NoStatus' = (Resolve-ParameterWithDefaultConfigurationValue -Name NoStatus -ConfigValueName DefaultNoStatus)
+        'NoStatus' = (Resolve-ParameterWithDefaultConfigurationValue `
+            -Name NoStatus -ConfigValueName DefaultNoStatus)
     }
 
-    return (Invoke-GHRestMethodMultipleResult @params |
-        Add-GitHubTeamAdditionalProperties)
+    $result = Invoke-GHRestMethodMultipleResult @params | Add-GitHubTeamAdditionalProperties
+
+    if ($PSBoundParameters.ContainsKey('TeamName'))
+    {
+        $team = $result | Where-Object -Property name -eq $TeamName
+
+        if ($null -eq $team)
+        {
+            throw "Team '$TeamName' not found"
+        }
+        else
+        {
+            $uriFragment = "/teams/$($team.Id)"
+            $description = "Getting team $($team.Id)"
```

Comment:
Technically it's a lower-case property:

```suggestion
            $uriFragment = "/teams/$($team.id)"
            $description = "Getting team $($team.id)"
```


The added one is SentenceCase: `$team.TeamId`